03/app.js

```diff
@@ -1,28 +1,24 @@
 export default randomNumber = (min, max) => {
-    if (areArgumentsValid(min, max)) {
-        const result = Math.random() * (max - min) + min;
+    if (isValidRange(min, max)) {
+        if (isMinValueValid(min) && isMaxValueValid(max)) {
```

Comment:
Funkcje `isMinValueValid` oraz `isMaxValueValid` robią to samo. Należałoby to inaczej rozwiązać:
```
if(!isValueValid(min)) {
  throw new Error(`Minimum value should be a number!`);
}

if(!isValueValid(max)) {
  throw new Error(`Maximum value should be a number!`);
}

```

PS. Należy pamiętaj, że `NaN` to też  number ;)

Reply:
Wprowadziłem poprawkę – mam nadzieję, że teraz będzie ok :).